src/tests/unit/findWithPagination.test.ts

```diff
@@ -48,39 +49,64 @@ afterEach(async () => {
   await connection.query("ROLLBACK").catch(() => {});
   jest.resetAllMocks();
 });
-
 describe("pagination", () => {
   beforeAll(async () => {
     const personRepository = connection.getRepository(Person);
     const ownerA = await personRepository.save({ name: "A" });
     const ownerB = await personRepository.save({ name: "B" });
 
+
     await connection.getRepository(Foo).insert([
-      { foo: "page-test-a", owner: ownerA },
-      { foo: "page-test-b", owner: ownerB },
-      { foo: "page-test-c", owner: ownerA },
+      { foo: "page-test-a", owner: ownerA, id: 'a' },
+      { foo: "page-test-b", owner: ownerB, id: 'b' },
+      { foo: "page-test-c", owner: ownerA, id: 'c' },
     ]);
   });
 
-  it("finds the first page with default order", async () => {
+/* Test suggestions
+  * Dry up the tests - one test for finding functionality, one test for ordering
+ default functionality, one test for ordering -no default option, a few for
+ *  pagination.
+ *  This way
+   we can assert on one expectation (of course maybe multiple assertions
+    still) per test.
+    *
+    * For example: Only one of the tests below needs to have a where clause
+    *  - and the first test should be split into two one for order and one
+    *  for pagination
+
+  * It might be nice to do the find test with number or boolean types to
+  avoid a like comparison on the where clause -- just to speed things up
+
+  * Given the time it would be ideal to mock some of the typeorm classes so
+  *  we can test the Outgoing class command messages to ensure they get sent.
+  *  For example, when we are testing that eager loading happens by default
+  *  we could instead assert the method `joinEagerRelations` is called on FindOptionsUtils
+ */
+  it("finds the first page with default order and default pagination", async () => {
+
     const fooRepository = connection.getRepository(Foo);
     const page = await findWithPagination(fooRepository, {
       select: ["foo"],
       where: { foo: Like("page-test-%") },
+      // loadEagerRelations: false
     });
-
-    expect(page.edges.length).toEqual(3);
+    console.log(page.edges[0].node)
+    expect(page.edges[0].node).toEqual({ foo: 'page-test-a' });
```

Comment:
With the previous single assertion the test asserted the pagination worked (partially), but not necessarily the default order 

I added one assertion below on line 95 to assert that the default ASC by id ordering worked

Then I replaced the assertion regarding length with two that I think tests the default pagination best (ie hasPreviousPage, and hasNextPage)